src/configurations/destinations/zoho/schema.json

```diff
@@ -0,0 +1,210 @@
+{
+  "configSchema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "required": ["region"],
+    "properties": {
+      "region": {
```

Comment:
will keep only US as option, because dynamic data fields depend on region. This way for now customer will not be confused and there will be no way to choose other options